styles.css

```diff
@@ -150,3 +145,28 @@
     padding-left: 5px;
     margin-top: 20px;
 }
+
+.qpy-show-more div {
+    text-align: justify;
+}
+.qpy-show-more .collapse:not(.show) {
+    display: block;
+    height: 4rem;
+    overflow: hidden;
```

Comment:
Hier könntest du noch das Folgende hinzufügen für einen fade-out Effekt ähnlich wie bei den Video-Kommentaren in ISIS:
```
    -webkit-mask-image: linear-gradient(to bottom, black 75%, transparent 100%);
    mask-image: linear-gradient(to bottom, black 75%, transparent 100%);
```
Probier gerne mal aus, ob das gut aussieht :) Den Wert `75%` musst du vielleicht noch anpassen.